test/reputation-system/root-hash-submissions.js

```diff
@@ -573,21 +574,38 @@ contract("Reputation mining - root hash submissions", accounts => {
       await badClient2.submitRootHash();
       await forwardTime(MINING_CYCLE_DURATION / 2, this);
 
+      const repCycle = await getActiveRepCycle(colonyNetwork);
+      const rewardIncrement = await repCycle.getDisputeRewardIncrement();
+
+      const blockBeforeChallenge = await currentBlock();
       await accommodateChallengeAndInvalidateHash(colonyNetwork, this, goodClient, badClient, {
         client2: { respondToChallenge: "colony-reputation-mining-increased-reputation-value-incorrect" }
       });
 
-      const invalidatorLock = await tokenLocking.getUserLock(clnyToken.address, accounts[0]);
-      expect(invalidatorLock.balance, "Account was not rewarded properly").to.eq.BN(new BN(invalidatorLockBefore.balance).add(MIN_STAKE.muln(2)));
+      const miner1NonceBefore = await web3.eth.getTransactionCount(MINER1, blockBeforeChallenge.number);
+      const miner1NonceAfter = await web3.eth.getTransactionCount(MINER1);
+
+      const miner2NonceBefore = await web3.eth.getTransactionCount(MINER2, blockBeforeChallenge.number);
+      const miner2NonceAfter = await web3.eth.getTransactionCount(MINER2);
+      const miner1SuccessfulDefences = miner1NonceAfter - miner1NonceBefore;
+      // The bad miner has one of these transactions that fails
+      const miner2SuccessfulDefences = miner2NonceAfter - miner2NonceBefore - 1;
```

Comment:
I'm sure the order of operations here is right but... a parentheses would help since subtraction is not associative...

Reply:
Prettier doesn't like it I'm afraid! The calculation is right though.